source/common/tracing/BUILD

```diff
@@ -39,3 +36,17 @@ envoy_cc_library(
         "//source/common/runtime:uuid_util_lib",
     ],
 )
+
+envoy_cc_library(
+    name = "lightstep_tracer_lib",
+    srcs = [
```

Comment:
Nit: Very slight preference for `srcs = ["foobar.cc"],` style when there is only one item in the list.